Library/Homebrew/extend/os/linux/cask/quarantine.rb

```diff
@@ -0,0 +1,26 @@
+# typed: strict
+# frozen_string_literal: true
+
+module OS
+  module Linux
+    module Cask
+      module Quarantine
+        extend T::Helpers
+
+        requires_ancestor { ::Cask::Quarantine }
+
+        sig { returns(Symbol) }
+        def self.check_quarantine_support
+          :linux
+        end
+
+        sig { returns(T::Boolean) }
+        def self.available?
+          false
+        end
```

Comment:
It's a stylistic choice until I get around to writing a linter rule, but I like these as endless `def`s, since they are pseudo-constants. (It also saves some vertical real estate.):
```suggestion
        def self.available? = false
```